src/test/java/com/edmunds/tools/databricks/maven/ImportWorkspaceMojoTest.java

```diff
@@ -56,18 +65,37 @@ public void testImportWorksWithProperPath() throws Exception {
     // Cannot use expected exceptions with a MojoExecutionException...
     @Test
     public void testImportFailsWithImproperPath() throws Exception {
+
+        ImportWorkspaceMojo underTest = getNoOverridesMojo(GOAL, "_fails_validation");
+
         File workspacePath = new File(this.getClass().getResource("/notebooks")
-            .getPath());
+                .getPath());
         underTest.validate = true;
         underTest.setSourceWorkspacePath(workspacePath);
-        Mockito.when(project.getGroupId()).thenReturn("com.edmunds.nottest");
-        Mockito.when(project.getArtifactId()).thenReturn("mycoolartifact");
-        underTest.setProject(project);
         try {
             underTest.execute();
         } catch (MojoExecutionException e) {
+            assertThat(e.getMessage(), containsString("JOB NAME VALIDATION FAILED [ILLEGAL VALUE]:\n" +
+                    "Expected: [failed-test] but found: [test]"));
             return;
         }
         fail();
     }
+
+    @Test
+    public void testImportWorksWithProperPathAndOverrides() throws Exception {
```

Comment:
nitpicky here, but in a different issue I propose doing {method}_{situation}_{expectedResult} test names. What do you think?

Reply:
That's probably a good idea moving forward.